### DISCUSSION TITLE
Using gr-satellites to validate an amateur receive only UHF ground station

### DISCUSSION THREAD

**Discussion by xjx** [daniestevez/gr-satellites#456] [Q&A]
Hello Daniel and Community!

I am new in the SDR field but already have gained some experience. I need to ask the following question:

1. For validating an amateur ground station, I would like to decode satellite transmissions using gr-satellites. 

2. This would require trying with at least one active satellite that is supported by gr-satellites.

3. **. Would you help me with suggestions on what satellite should I pick that complies with the previous statement?**

4. The “more active” the satellite is, the better for catching that transmission and speed up the ground station validation process.

5. Besides the satellite being active, it would be interesting to listen for satellites that have powerful beacon transmissions, as well as with high repetition rate of those beacons.

6. I may add that I ask for your help after having tried all the above with no success so far. That is, I am correctly receiving and recording signals, but still have not been able to decode them.

I would really appreciate your help.

**Comment by OE3ALA** [comment]
Hi!

For my UHF ground station for observing PEGASUS, I have looked for satellites with the same transmitter mode (GFSK9k6) to validate the functionallity of gr-satellites. This was necessary back than, because our satellites wasn't sending at that time. 

I can recommend following satellites, which should be demodulated by gr-satellites and fullfill your requirements 1, 2, 3 and 5:

- GRBAlpha ([Observations](https://network.satnogs.org/observations/?norad=47959))
- CUTE ([Observations](https://network.satnogs.org/observations/?norad=49263))
- BDSAT-2 ([Observations](https://network.satnogs.org/observations/?norad=55098)) *not in the list of supported satellites!
- Geoscan-Edelveis ([Observations](https://network.satnogs.org/observations/?norad=53385))

Some words about PEGASUS ([Observations](https://network.satnogs.org/observations/?norad=42784)). PEGASUS is supported by gr-satellites, at the end of its lifetime and (i guess) only sending when its solar panels are in sunlit. But you can also try to observe it! I am currently receiving and demodulating its signals daily on our [UHF Ground Station](https://network.satnogs.org/observations/?station=3045) in Austria. For this satellite I can tell you, if the demodulated data packages are as expected.

You find all supported satellites of gr-satellites here: [Link](https://gr-satellites.readthedocs.io/en/latest/supported_satellites.html)
I have installed gr-satellites according to [this instructions](https://github.com/kng/satnogs_gr-satellites/blob/main/INSTALL.md) for satnogs_gr-satellites (installing gr-satellites from build according to the documentation didn't work out for me, I guess i have chosen the wrong GNU Radio version).

When you observe satellites, also have an eye on the success rates of the satellites for the specific transmitter. You find all this information in the SatNOGS Network, when you schedule the satellite.

Cheers!

73 de
Alex, oe3ala
